README.md

```diff
@@ -1 +1,3 @@
-# messenger_dev
```

Comment:
Put a ticket in to populate this readme with architecture description and instructions